dev/bin/localconfig-example

```diff
@@ -10,6 +10,10 @@ MANAGED_CONNECTORS_AUTH_OFFLINE_TOKEN=my-offline-token
 
 # RHOAS_USERNAME=
 
+# This will configure the local RHOSE instance to use your username as prefix for Kafka topics and Managed Connectors
+# making it way easier to identify who is using it.
+EVENT_BRIDGE_RESOURCE_PREFIX=my_username
```

Comment:
```suggestion
export EVENT_BRIDGE_RESOURCE_PREFIX=my_username
```

Reply:
AFAIK export is needed as the `EVENT_BRIDGE_RESOURCE_PREFIX` is not a script property but env variable applied directly as manager config.

Reply:
yes that's true...good catch @sutaakar :+1:

Reply:
hmm, when thinking about it again, maybe it is not needed.....
not sure whether the variables specified in localconfig are applied as env variables
I just followed the email instructions, which were mentioning export
may be worth trying without export